### ISSUE TITLE
Is there a way to manually insert page breaks?

### ISSUE THREAD

**Issue by keithrz** [rkusa/pdfjs#39]
I am looking for something like
```
doc.pageBreak();
```
or
```
doc.addPage();
```
I would expect to add it at the document level, or as an op:
```
var ops = doc.ops();
ops.op('pgbr');
```
If it is an op, I'm not sure where I can find that information.  Is there another website that shows all possible "pdf operations" ?  If so, could you please link to it in your documentation?

Thanks for the great tool!!

**Comment by rkusa** [comment]
Sounds like an essential feature, I've put onto my todo list. Should be a no-brainer.

You can find the complete operator reference in the [PDF Reference](http://www.adobe.com/content/dam/Adobe/en/devnet/acrobat/pdfs/pdf_reference_1-7.pdf). At page **196** you'll get an overview table that can be used to jump between the different operator categories. Unfortunately, there is no page break operation.

Thanks for your feedback.

**Comment by keithrz** [comment]
Thank you for the quick response!  I was just starting to look at that pdf reference document; thanks so much for pointing out which page it was on (out of 1300+ pages) !!

**Comment by rkusa** [comment]
I've added a `.pageBreak()` method. Thanks for the suggestion

**Comment by keithrz** [comment]
Awesome!  I'll test it out in what I assume will be 1.0.0-alpha.5.  Thanks!!

**Comment by rkusa** [comment]
I've published `1.0.0-alpha.5`. Please let me know if you encounter any issue
